create-stack/test/acceptance/full_test.go

```diff
@@ -128,6 +130,18 @@ func testFull(cliPath string) func(*testing.T, spec.G, spec.S) {
 			Expect(err).NotTo(HaveOccurred())
 			Expect(runReleaseDate).To(BeTemporally("~", time.Now(), 10*time.Minute))
 			Expect(buildReleaseDate).To(Equal(runReleaseDate))
+
+			cmd = exec.Command(
+				"docker", "run", "--rm",
+				settings.Build.BaseRef,
+				"git", "config", "--system", "--list",
+			)
+			output, err = cmd.CombinedOutput()
+			Expect(err).NotTo(HaveOccurred(), fmt.Sprintf("failed to run container: %s", output))
+
+			Expect(string(output)).To(ContainSubstring("safe.directory=/workspace"))
+			Expect(string(output)).To(ContainSubstring("safe.directory=/workspace/source-ws"))
+			Expect(string(output)).To(ContainSubstring("safe.directory=/workspace/source"))
```

Comment:
Should the tiny tests have these assertions as well since it works off of the same build image?

Reply:
Updated to include the tiny tests.